airflow/bin/cli.py

```diff
@@ -607,6 +615,11 @@ class CLIFactory(object):
             ("--stdout", ), "Redirect stdout to this file"),
         'log_file': Arg(
             ("-l", "--log-file"), "Location of the log file"),
+        'yes': Arg(
+            ("-y", "--yes"),
+            "Do not prompt to confirm. Use with care!",
+            "store_true",
```

Comment:
What does store_true mean?

Reply:
Current style seems to put lines 619-621 on one line.